cmd/swarmctl/cluster/inspect.go

```diff
@@ -52,6 +53,28 @@ func printClusterSummary(cluster *api.Cluster) {
 			fmt.Fprintf(w, "    %s: %s\n", ca.Protocol, ca.URL)
 		}
 	}
+
+	if cluster.Spec.DefaultLogDriver != nil {
+		fmt.Fprintf(w, "DefaultLogDriver\t: %s\n", cluster.Spec.DefaultLogDriver.Name)
+		var keys []string
+
+		if cluster.Spec.DefaultLogDriver.Options != nil {
```

Comment:
Should this be `if len(cluster.Spec.DefaultLogDriver.Options) != 0`

Also, don't you need to make sure `DefaultLogDriver` is non-nil to avoid a potential nil pointer dereference?